docs/reference/search/search-vector-tile-api.asciidoc

```diff
@@ -0,0 +1,760 @@
+[[search-vector-tile-api]]
+=== Search vector tile API
+++++
+<titleabbrev>Search vector tile</titleabbrev>
+++++
+
+experimental::[]
+
+Searches a vector tile for geospatial values. Returns results as a binary
+https://docs.mapbox.com/vector-tiles/specification[Mapbox vector tile].
+
+////
+[source,console]
+----
+PUT my-index
+{
+  "mappings": {
+    "properties": {
+      "my-geo-field": {
+        "type": "geo_point"
+      }
+    }
+  }
+}
+
+PUT my-index/_doc/0?refresh
+{
+  "my-geo-field": "37.3864953,-122.0863176"
+}
+----
+////
+
+[source,console]
+----
+GET my-index/_mvt/my-geo-field/15/5271/12710
+----
+// TEST[continued]
+
+[[search-vector-tile-api-request]]
+==== {api-request-title}
+
+`GET <target>/_mvt/<field>/<zoom>/<x>/<y>`
+
+`POST <target>/_mvt/<field>/<zoom>/<x>/<y>`
+
+[[search-vector-tile-api-prereqs]]
+==== {api-prereq-title}
+
+* Before using this API, you should be familiar with the
+https://github.com/mapbox/vector-tile-spec[Mapbox vector tile specification].
+
+* If the {es} {security-features} are enabled, you must have the `read`
+<<privileges-list-indices,index privilege>> for the target data stream, index,
+or alias.
+
+[[search-vector-tile-api-path-params]]
+==== {api-path-parms-title}
+
+`<target>`::
+(Required, string) Comma-separated list of data streams, indices, or aliases to
+search. Supports wildcards (`*`). To search all data streams and indices, omit
+this parameter or use `*` or `_all`.
+
+`<field>`::
+(Required, string) Field containing geospatial values to return. Must be a
+<<geo-point,`geo_point`>> or <<geo-shape,`geo_shape`>> field.
```

Comment:
Maybe we should explicitly say that fields should be indexed with doc values and cannot be nested fields?

Reply:
Thanks for the suggestion. I added this with 68dabc1fbdc3acc6ba2de6573de97b8fed175914.